src/components/LoanCards/Buttons/LoanSelectedText.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+	<div class="loan-selected-text small-text">
+		Another lender has selected this loan. Please choose a different borrower to support.
```

Comment:
Where did this copy come from? I don't think it represents the actual situation (a funded loan) very well.